README.md

```diff
@@ -19,4 +19,5 @@ For more information, please see the links below.
 
```

Comment:
While not directly related to the changes, I noticed that the README ends abruptly after the FOSSA status badge. Consider adding a conclusion or a call-to-action section to make the README more engaging and informative.
```suggestion
# After the FOSSA status badge, add:

## Contributing

We welcome contributions to this project! Please see our [Contributing Guidelines](CONTRIBUTING.md) for more information on how to get started.

## License

This project is licensed under the [MIT License](LICENSE). For full details, see the LICENSE file in the repository.

---

Thank you for your interest in our project. If you have any questions or need further assistance, please open an issue or contact our team.
```